apps/web/src/styles/globals.css

```
@@ -1,7 +1,8 @@
 @import "tailwindcss";
 
-/* Scan route files for Tailwind utility classes */
+/* Scan route and component files for Tailwind utility classes */
 @source "../routes";
```

Stylelint (17.4.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)